src/twisted/internet/test/test_tcp.py

```diff
@@ -122,7 +125,16 @@
 
     getLinkLocalIPv6Addresses = _win32ifaces.win32GetLinkLocalIPv6Addresses
 
+    try:
+        from twisted.internet.iocpreactor.reactor import IOCPReactor
+    except ImportError:  # pragma: no cover
+        # On our CI, we always have iocp-support package available.
+        # This is here for the case in which someone runs the tests on
+        # Windows, without the package installed.
+        IOCPReactor = None  # type: ignore[misc,assignment]
```

Comment:
Hmm. Seeing this comment written out here, it occurs to me that this *is* a case we might want to care about? At least at one point, the constituency of Windows users who could install Twisted but could not compile (or even necessarily install) native code was one we cared about.  If we do care about them, we should probably have CI; if not, we should probably just delete this code.

However that is a long and potentially annoying policy discussion and for this PR the coverage exception is fine, I don't want to get bogged down here, just raise the issue for addressing later.